src/App.test.js

```diff
@@ -715,4 +716,50 @@ describe('App component', () => {
       expect(localStorageItems['dopplerFirstOrigin.value']).toBeUndefined();
     });
   });
+
+  describe('activation in progress parameter', () => {
+    it('should call google adwords', () => {
+      // Arrange
+      const dependencies = {
+        window: {
+          gtag: jest.fn(),
+          Menubutton: MenubuttonFake,
+        },
+      };
+
+      // Act
+      render(
+        <AppServicesProvider forcedServices={dependencies}>
+          <Router initialEntries={['/login?activationInProgress%20=%20true']}>
+            <App locale="en" />
+          </Router>
+        </AppServicesProvider>,
+      );
+
+      // Assert
+      expect(dependencies.window.gtag).toBeCalledTimes(1);
+    });
+
+    it('should not to be call google adwords', () => {
```

Comment:
Could you include the condition?

```suggestion
    it('should not to be call google adwords when `activationInProgress` is not present in the query string', () => {
```